bin/elasticurl/main.c

```diff
@@ -453,6 +461,10 @@ static void s_on_signing_complete(struct aws_http_message *request, int error_co
 
 static void s_on_client_connection_setup(struct aws_http_connection *connection, int error_code, void *user_data) {
     struct elasticurl_ctx *app_ctx = user_data;
+    if (aws_http_connection_get_version(connection) != app_ctx->required_http_version) {
```

Comment:
how does required_http_version get set if user doesn't specify one explicitly?

Reply:
I forgot about this case.🤦‍ Fixed.